.github/workflows/build.yml

```diff
@@ -0,0 +1,324 @@
+      - uses: jdx/mise-action@c37c93293d6b742fc901e1406b8f764f6fb19dac # v2.4.4
+        with:
+          version: 2025.7.12
+      - &checkout-build-logic
+        name: Checkout build logic
+        run: |
+          git submodule update --init --depth 1 -- build-logic/common
+      - &create-gradle-user-home
+        name: Create Gradle User Home
+        shell: bash
+        run: |
+          export GRADLE_USER_HOME=${GITHUB_WORKSPACE}/.gradle
+          mkdir -p ${GRADLE_USER_HOME}
+          echo "GRADLE_USER_HOME=${GRADLE_USER_HOME}" >> $GITHUB_ENV
+          export TODAY=$(date '+%Y-%m-%d')
+          echo "TODAY=${TODAY}" >> $GITHUB_ENV
+          find . -name '*.gradle.kts' -type f -exec md5sum {} \; | sort && md5sum gradle/libs.versions.toml && md5sum gradle/wrapper/gradle-wrapper.properties && md5sum gradle.properties > gradle-md5-sums.txt 
+          export GRADLE_CACHE_KEY=$(md5sum gradle-md5-sums.txt | awk '{ print $1 }')
+          echo "GRADLE_CACHE_KEY=${GRADLE_CACHE_KEY}" >> $GITHUB_ENV
+          rm gradle-md5-sums.txt
+      - &cache-gradle-dependencies
+        name: Cache Gradle Dependencies
+        uses: SonarSource/ci-github-actions/cache@v1
+        with:
+          path: ${{ env.GRADLE_USER_HOME }}
+          key: gradle-${{ env.GRADLE_CACHE_KEY }}
+      - uses: SonarSource/ci-github-actions/build-gradle@v1
+        with:
+          deploy-pull-request: true
+          artifactory-reader-role: public-reader
+          artifactory-deployer-role: public-deployer
+          # issue in build-gradle action, skipping tests manually in gradle-args
+          skip-tests: true
+          # There might be a warning: Warning: Failed to fetch short-lived token for Develocity
+          # it will be addressed in BUILD-8926
+          use-develocity: true
+          gradle-args: -x sonar storeProjectVersion
+
+  build_test_analyze:
+    needs: [build]
+    runs-on: sonar-l-public
+    name: Build Test Analyze
+    permissions:
+      id-token: write
+      contents: write
+    steps:
+      - uses: actions/checkout@08eba0b27e820071cde6df949e0beb9ba4906955 # v4.3.0
+      - uses: jdx/mise-action@c37c93293d6b742fc901e1406b8f764f6fb19dac # v2.4.4
+        with:
+          version: 2025.7.12
+      - *checkout-build-logic
+      - *create-gradle-user-home
+      - *cache-gradle-dependencies
+      - uses: SonarSource/ci-github-actions/build-gradle@v1
+        with:
+          deploy-pull-request: false
+          artifactory-reader-role: public-reader
+          artifactory-deployer-role: qa-deployer
+          skip-tests: false
+          use-develocity: true
+          gradle-args: -x artifactoryPublish
+      - name: Upload test results
+        if: always() && ! cancelled()
+        uses: actions/upload-artifact@6f51ac03b9356f520e9adb1b1b7802705f340c2b # v4.5.0
+        with:
+          name: test-results
+          path: '**/test-results/**/*.xml'
+
+  qa_os_win:
+    needs: [build]
+    runs-on: windows-latest-large
```

Comment:
I think this is the old name, I found [reference](https://xtranet-sonarsource.atlassian.net/wiki/spaces/Platform/pages/4252827657/BUILD-8835+-+2025+Q3+Standardize+GitHub+Runner+Groups+and+Labels) that now it should be `github-windows-latest-l`. (It seems the doc for AI was not updated)
Also, I can see that `github-windows-latest-s` is already 8 CPU and 32 Go of RAM, do we really need the `l` version? (couldn't find the spec, but if it follow ubuntu pattern, it would be like 16 CPU and 64 Go of RAM).